docs/01-Using-Fleet/03-REST-API.md

```diff
@@ -1056,6 +1056,38 @@ Request (`filters` is specified):
 
 `Status: 200`
 
+### Get host device mapping
+
+Returns the mapping of users' email addresses to the host, as reported by optional osquery extensions such as the Google Chrome profiles.
+
+`GET /api/v1/fleet/hosts/{id}/device_mapping`
+
+#### Parameters
+
+| Name       | Type              | In   | Description                                                                   |
+| ---------- | ----------------- | ---- | ----------------------------------------------------------------------------- |
+| id         | integer           | path | **Required**. The host's `id`.                                                |
+
+#### Example
+
+`GET /api/v1/fleet/hosts/1/device_mapping`
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "device_mapping": [
+    {
+      "host_id": 1,
```

Comment:
Do we need the `host_id`? If so, let's update #400's description to include `host_id`.

Reply:
Not strictly speaking, but I think as a general rule it's good to be able to identify what entity the payload is associated with, though arguably in this case the host_id could be hoisted to a field on the top-level object instead of repeated for each mapping. I think I'll adjust it this way, and will mention it in the ticket.